airbyte-integrations/connectors/source-google-sheets/source_google_sheets/source.py

```diff
@@ -149,6 +149,7 @@ def _read(
         catalog: ConfiguredAirbyteCatalog,
     ) -> Generator[AirbyteMessage, None, None]:
         client = GoogleSheetsClient(self.get_credentials(config))
+        client.Backoff.row_batch_size = config["batch_size"]
```

Comment:
```suggestion
        client.Backoff.row_batch_size = config.get("batch_size", 200)
```
for old configs without batch_size.

Reply:
Fixed